src/scripts/saveToOneNote/oneNoteApiWithLogging.ts

```diff
@@ -90,13 +89,12 @@ export class OneNoteApiWithLogging implements OneNoteApi.IOneNoteApi {
 			let event = new Log.Event.PromiseEvent(eventLabel);
 			let correlationId: string;
 			return func().then((response) => {
+				event.setCustomProperty(Log.PropertyName.Custom.CorrelationId, response.request.getResponseHeader(Constants.HeaderValues.correlationId));
```

Comment:
(As I understand this code change, we will no longer set correlationid on the header) If we time out, you can still use sessionid to look for logs, so this is good.